### DISCUSSION TITLE
How to open a file in editor using theia.

### DISCUSSION THREAD

**Discussion by kumbham** [eclipse-theia/theia#9645] [General] [answered]
I would like to know how to open a file in the editor when launching theia. 
ex: I have `samplefile.txt` in my workspace directory i.e `/home/user/sample_space`. I would like to launch theia with samplefile.txt opened in the editor. 

I have tried passing this information in URL as `http://localhost:3000/#/home/users/sample_space/samplefile.txt`. Theia throws an `invalid workspace file` error.

Any help would be appreciated, thanks!

**Comment by vince-fugnitto** [comment, marked as answer]
@kumbham thank you for the discussion, it is not yet supported to open a file directly from the URL as you’ve described: https://github.com/eclipse-theia/theia/issues/4604.
